src/components/item-blog.tsx

```diff
@@ -6,7 +6,11 @@ import { getDate } from "./utils"
 export const ItemBlog = ({ data }) => {
 
     const [focused, changeFocused] = useState(false);
-
+    const authors = []
+    const authorObjects = Object.entries(data.frontMatter.authors)
+    for (var author of authorObjects) {
+        authors.push(<small className="pl-2 font-sans">{author[1]['name']}</small>)
+    }
```

Comment:
I think we want to print out authors, comma separated here, right?
`{author[1]['name']}` gets the 2nd index of something unless I'm missing something here.

Reply:
This does print out the author names (see the second screenshot above), but I'd be happy to reformat

Reply:
@phixMe refactored item-blog to print comma-separated values from an array of author names instead of a second index. It also prints comma-separated strings in bold now (to distinguish the author names from the other date more clearly).